config/config.go

```diff
@@ -38,6 +38,8 @@ type Config struct {
 	SolanaIndexerWorkers       int
 	SolanaIndexerRetryInterval time.Duration
 	CommsMessagePush           bool
+	AcdcChainID                uint
```

Comment:
Acdc?!?

Reply:
It's what they are called in audiusd, I will ask @alecsavvy to defend this lol.

Reply:
I suppose I can rename _our_ config variables though.